### PR TITLE
Fix misplaced entry on CHANGELOG.md

### PR DESCRIPTION
**Pull request by giubacc**
Signed-off-by: Giuseppe Baccini <giuseppe.baccini@suse.com>

# Describe your changes

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Expose `ETag` header in the Traefik s3gw ingress to allow multi-part
   uploads via browser (gh#aquarist-labs/s3gw#170).
+- build containers based on openSUSE Leap 15.4 instead of Tumbleweed.
```
Comment (votdev) [inline CHANGELOG.md:14]:
```suggestion
- Build containers based on openSUSE Leap 15.4 instead of Tumbleweed.
```

**Review by votdev — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (giubacc) [reply CHANGELOG.md:14]: All previous entries begin lowercase; maybe it could be more correct fix Expose -> expose ?

Comment (jecluis) [reply CHANGELOG.md:14]: Please don't get caught up in those details. There will always be some entries in lowercase and others in uppercase. What matters is that they are there.

**Review by jecluis — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (votdev) [reply CHANGELOG.md:14]: If what we have done so far has been wrong, then we should not take this as an opportunity to continue doing it wrong. We should pay attention to correct spelling.

Comment (giubacc) [reply CHANGELOG.md:14]: Personally I totally agree on this; I would only do this on a separate PR, fixing all entries.
Given that the current convention (wrong or right) was to have all entries begin with a lowercase, I would say that the right thing to do is to keep that convention for consistency.
But I repeat, no problem if we want to change this.

**Pull request merged**